web/src/giphy.ts

```diff
@@ -58,29 +64,26 @@ function get_rating() {
     return "g";
 }
 
-async function renderGIPHYGrid(targetEl) {
-    const {renderGrid} = await import(/* webpackChunkName: "giphy-sdk" */ "@giphy/js-components");
-    const {GiphyFetch} = await import(/* webpackChunkName: "giphy-sdk" */ "@giphy/js-fetch-api");
```

Comment:
I removed the dynamic imports as they don't work with typescript configuration setup currently. I added normal imports instead. What are the better ways to tackle this issue well ?

Reply:
Can you detail what doesn't work?

Reply:
It throws this error : _" Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node16', or 'nodenext' "_
This is a relevant issue related to this topic which i found in the typescript repo : [Error when trying to use dynamic import with module: "es2015"](https://github.com/Microsoft/TypeScript/issues/16675)

Reply:
Apparently we need to set the module flag in the `tsconfig.json`  to one of those if we want to continue using dynamic imports. I fear if the change will break any other part of code or not.